internal/providers/docker/components/network/lifecycle.go

```diff
@@ -82,7 +83,12 @@ func (n *Network) Dispose(ctx context.Context, input *core.DisposeInput) (*core.
 	if n.NetworkID == "" {
 		return &core.DisposeOutput{}, nil
 	}
-	if err := n.Docker.NetworkRemove(ctx, n.NetworkID); err != nil {
+	err := n.Docker.NetworkRemove(ctx, n.NetworkID)
+	if docker.IsErrNotFound(err) {
+		n.Logger.Infof("disposing network not found: %q", n.NetworkID)
```

Comment:
Nit but I ran into this before I saw this PR and didn't understand what a "disposing volume" was. Would prefer something like `volume to be removed cannot be found`.

Reply:
ok fixed in bb52c7e